content/docs/operating/integrations.md

```diff
@@ -97,6 +97,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [SIGNL4](https://www.signl4.com/blog/portfolio_item/prometheus-alertmanager-mobile-alert-notification-duty-schedule-escalation)
   * [SMS](https://github.com/messagebird/sachet): supports [multiple providers](https://github.com/messagebird/sachet/blob/master/examples/config.yaml)
   * [SNMP traps](https://github.com/maxwo/snmp_notifier)
+  * [STOMP](https://github.com/thewillyhuman/alertmanager-stomp-forwarder)
```

Comment:
T comes after Q in the alphabet, so this should be one line further down. Otherwise, :+1:

Reply:
🥲 Did not notice the following link, sorry very much. Should be fixed now.